modal/exception.py

```diff
@@ -22,6 +22,19 @@ class Error(Exception):
     """
 
 
+class _ContainerException(Exception):
```

Comment:
I prefer this name a little more:

```suggestion
class _WrappedException(Exception):
```

At a glance, I thought "Container" meant "in the runtime container".